### PR TITLE
added uptime module

### PR DESCRIPTION
**Pull request by MagiCrazy**
Hi,
I added an uptime module I would like to share.
Tell me if everything's right.

### CODE REVIEW

File: config/goblocks-full.yml [hunk 1]
```diff
@@ -120,6 +120,10 @@ blocks:
       label: "V: "
       update_signal: 1
 
+    - type: Uptime
```
Comment (davidscholberg) [inline config/goblocks-full.yml]: This should be lowercase.

**Review by davidscholberg — CHANGES_REQUESTED** [review]
Thank you for this pull request! It looks great. There's just a small typo in the example config, but otherwise the code is fine.

**Comment by MagiCrazy** [comment]
That's OK for me =)

**Comment by MagiCrazy** [comment]
up! is there a problem with my fix?

**Comment by davidscholberg** [comment]
Sorry, I've been busy lately. I'll try to get to this sometime this week.

**Review by davidscholberg — APPROVED** [review]

**Comment by davidscholberg** [comment]
Thank you, @MagiCrazy.

I think it would be nice if there was a config option to specify a time format to use for the uptime module, just like the time module does. If you have any interest in adding that feature, please let me know. Otherwise, I'll probably add it myself.

**Comment by MagiCrazy** [comment]
Great idea, I'll add it when I have some time too, if you didn't already.

**Comment by davidscholberg** [comment]
Go for it :)

**Comment by davidscholberg** [comment]
@MagiCrazy I looked into this a little, and it turns out that Go doesn't seem to have built-in handling for formatting durations, which makes this feature somewhat non-trivial to implement. If you haven't already started, don't worry about doing this feature. I'll take care of it.

**Comment by MagiCrazy** [comment]
@davidscholberg I looked into it too, and went to the same conclusion.

**Comment by davidscholberg** [comment]
@MagiCrazy FYI, I added custom formatting for durations in commit a02fcfce91cfb0e7b5830dcd76b6ea9ba65f1378.

**Pull request merged**